tests/lib/rules/valid-jsdoc.js

```diff
@@ -429,6 +429,14 @@ ruleTester.run("valid-jsdoc", rule, {
                 type: "Block"
             }]
         },
+        {
```

Comment:
I also thought about what you saying. I think to correctly manage that we need an option `requireReturnType` by default set to `true`. This way we can do non-breaking change.